src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -319,7 +398,7 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdID, uint nCmdexecopt, IntPtr pv
             if (pguidCmdGroup == ReSharperCommandGroup && nCmdID >= ResumeId && nCmdID <= ToggleSuspendId)
             {
                 // Don't delay command processing to update resharper status
-                Task.Run(() => InvokeBelowInputPriorityAsync(UpdateStateMachine));
+                StartUpdateStateMachine(pollForStatus: false);
```

Comment:
is poolForStatus perf optimization? any harm on not doing this?

Reply:
Regarding our discussion, this isn't needed if I use the cancellationToken instead.  So I'll remove it.